staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -899,6 +903,16 @@ func (az *Cloud) reconcileLoadBalancerRule(
 		ports = []v1.ServicePort{}
 	}
 
+	enableTCPReset := false
+	if az.useStandardLoadBalancer() {
+		if v, ok := service.Annotations[ServiceAnnotationLoadBalancerEnableTCPReset]; ok {
+			klog.V(2).Infof("reconcileLoadBalancerRule lb name (%s) flag(%s) is set to %s", lbName, ServiceAnnotationLoadBalancerEnableTCPReset, v)
+			if strings.EqualFold(v, "true") {
+				enableTCPReset = true
+			}
+		}
+	}
```

Comment:
Return error when basic load balancer is used here. Failing silently without errors would confuse users.

Reply:
Fixed.